src/Routing/Types.purs

```diff
@@ -0,0 +1,6 @@
+module Routing.Types where
+
+import qualified Data.StrMap as M
+
+data RoutePart = Path String | Query (M.StrMap String)
+type Route = [RoutePart]
```

Comment:
Adding support of multiple queries is easy, and it can be useful sometimes. 
I.e. in producing parameterized ajax and filtering grid with many filters simultaneously.